src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -399,10 +399,14 @@ REPORT_CACHE = 'redis'
 
 redis_cache = {
     'BACKEND': 'django_redis.cache.RedisCache',
+{% set redis_url = 'redis://:' ~ localsettings.REDIS_HOST ~ ':' ~ localsettings.REDIS_PORT %}
+{% if localsettings.get('REDIS_PASSWORD') %}
+{% set redis_url = 'redis://:' ~ localsettings.REDIS_PASSWORD ~ '@' ~ localsettings.REDIS_HOST ~ ':' ~ localsettings.REDIS_PORT %}
```

Comment:
From the docs: 
```
~
Converts all operands into strings and concatenates them.

{{ "Hello " ~ name ~ "!" }} would return (assuming name is set to 'John') Hello John!.
```

Reply:
How did i manage to run it first time on initial PR ?

Reply:
great question. Did you not run it prior to this change? When it was using nested if statements?

Reply:
i ran it when it was like this https://github.com/dimagi/commcare-cloud/pull/3658/files#diff-6b0c50c559a034e6ce7b33045ddf088eR402-R410

Reply:
wait , now that i think about it . since you made the changes i might not have pulled it locally.